src/test/java/org/entando/kubernetes/utils/TenantContextJunitExt.java

```diff
@@ -4,17 +4,23 @@
 
 import lombok.extern.slf4j.Slf4j;
 import org.entando.kubernetes.config.tenant.thread.TenantContextHolder;
+import org.junit.jupiter.api.extension.AfterAllCallback;
 import org.junit.jupiter.api.extension.BeforeAllCallback;
 import org.junit.jupiter.api.extension.ExtensionContext;
 
 
 @Slf4j
-public class TenantContextJunitExt implements BeforeAllCallback {
+public class TenantContextJunitExt implements BeforeAllCallback, AfterAllCallback {
 
     @Override
     public void beforeAll(ExtensionContext extensionContext) throws Exception {
-
-        log.info("Setting tenant code within the context");
+        log.info("Setting primary tenant code within the context");
         TenantContextHolder.setCurrentTenantCode("primary");
     }
+
+    @Override
```

Comment:
@firegloves I added a clean after all, this was useful to find tests that need to use this junit extension, what do you think?